src/templates/static/why_us.jsx

```diff
@@ -40,10 +40,10 @@ const WhyUs = () => (
                     <div className='gr-row'>
                         <div className='gr-7 gr-12-m'>
                             <ul className='checked'>
-                                <li>{it.L('Real-time prices, benchmarked against interbank options markets.')}</li>
-                                <li>{it.L('Returns above 100% on simple <a href=\'[_1]\'>rise/fall</a> contracts.', it.url_for('trading?market=forex&formname=risefall'))}</li>
-                                <li>{it.L('Returns above 1000% on <a href=\'[_1]\'>higher/lower</a> and <a href=\'[_2]\'>touch/no-touch</a> trades.', it.url_for('trading?market=forex&formname=higherlower'), it.url_for('trading?market=forex&formname=touchnotouch'))}</li>
-                                <li>{it.L('No hidden fees or commissions.')}</li>
+                                <li>{it.L('Real-time prices, benchmarked against interbank options markets')}</li>
+                                <li>{it.L('High returns on [_1]rise/fall[_2] contracts', `<a href='${it.url_for('trading?market=forex&formname=risefall')}'>`, '</a>')}</li>
+                                <li>{it.L('Returns above 1000% on [_1]higher/lower[_2] and [_3]touch/no-touch[_4] trades.', `<a href='${it.url_for('trading?market=forex&formname=higherlower')}'>`, '</a>', `<a href='${it.url_for('trading?market=forex&formname=touchnotouch')}'>`, '</a>')}</li>
```

Comment:
we can reuse placeholder [_2] here instead of [_4] since they are the same element `</a>`